OpenRA.Mods.Common/Traits/Passenger.cs

```diff
@@ -38,6 +38,9 @@ public class PassengerInfo : ITraitInfo
 		[Desc("Range from self for looking for an alternate transport (default: 5.5 cells).")]
 		public readonly WDist AlternateTransportScanRange = WDist.FromCells(11) / 2;
 
+		[Desc("What have stance of target(Ally, Neutral, Enemy). (default: Ally)")]
```

Comment:
Would `ValidStances` work for everyone?

We use this in other targeting code and this is not too different in this regard.